src/ui/src/images/index.ts

```diff
@@ -0,0 +1,5 @@
+import PagerDutyLogo from './PagerDutyLogo';
```

Comment:
Is there a reason to go from tsx to ts?

Reply:
because the index file is handling imports and will never contain `tsx` syntax, hence not necessary